apps/channels/models.py

```diff
@@ -43,11 +43,24 @@ def form(self, team: Team):
 
     def extra_form(self, *args, **kwargs):
         from apps.channels import forms
+        from apps.service_providers.models import MessagingProviderType
+
+        channel = kwargs.pop("channel", None)
 
         match self:
             case self.TELEGRAM:
                 return forms.TelegramChannelForm(*args, **kwargs)
             case self.WHATSAPP:
+                if channel and channel.messaging_provider.type == MessagingProviderType.turnio:
+                    webhook_url = absolute_url(
+                        reverse("channels:new_turn_message", kwargs={"experiment_id": channel.experiment.public_id}),
+                        is_secure=True,
+                    )
+                    initial = kwargs.get("initial", {})
+                    initial.setdefault("verify_token", str(uuid.uuid4()))
```

Comment:
what is `verify_token`? I don't see it used anywhere else. Maybe just a copy paste?

Reply:
Yes, a copy and paste :sweat_smile: